ifrs17/Import/ImportScopeCalculation.ipynb

```diff
@@ -641,14 +641,14 @@
                 "\n    private PeriodType periodType => GetStorage().GetPeriodType(Identity.AmountType, Identity.EstimateType); ",
                 "\n    ",
                 "\n    static ApplicabilityBuilder ScopeApplicabilityBuilder(ApplicabilityBuilder builder) =>",
-                "\n        builder.ForScope<DiscountedCashflow>(s => s.WithApplicability<DiscountedCreditRiskCashflow>(x => x.Identity.Id.IsReinsurance && x.Identity.AmountType == AmountTypes.CDR));",
-                "\n",
-                "\n    [NotVisible]",
-                "\n    string EconomicBasis => GetContext();",
+                "\n        builder.ForScope<DiscountedCashflow>(s => s.WithApplicability<DiscountedCreditRiskCashflow>(x => x.Identity.Id.IsReinsurance && x.Identity.AmountType == AmountTypes.CDR)",
```

Comment:
This applicability can now be moved to the Nominals Scope. 
Here the Discound factor should be the right one given the changes in the GetYearlyYieldCuve.

Reply:
Should I get rid of the DiscountedCashflow scope completely? If I move both DiscountedCreditRiskCashflow and NEW_SCOPE, there is nothing else left.

Reply:
@dcolleoni Davide, do you mean DiscountedCreditRiskCashflow applicability? I think, you spotted this applicability accidentally. Or was it meant for the NEW_SCOPE that I had over there right below this applicability? If yes, that issue is already resolved now.

Reply:
I don't mean to change code that was present already. I mean the new code about the BOP,I  as we also discussed about.

Reply:
Done. The applicability is moved to the NominalCashflow scope.